File: evidence/generated-systems/angular/aerospace-on-angular/src/app/models/MaintenanceWorkOrder.js

// Define collection and schema for MaintenanceWorkOrder
export interface MaintenanceWorkOrder {
    workOrderNumber:
	type : string
    Aircraft:
	type : Schema.Types.ObjectId
    AirworthinessDirective:
	type : Schema.Types.ObjectId
    ServiceBulletin:
	type : Schema.Types.ObjectId
    Status:
 	type : String
#
    collection: 'maintenanceWorkOrders'
}

File: evidence/generated-systems/rubyonrails/aerospace-on-rails/aerospaceonrails/app/models/maintenanceworkorder.rb
class MaintenanceWorkOrder < ApplicationRecord
  enum Status: [:Open, :InProgress, :AwaitingParts, :Closed, :Deferred]


  has_many :Aircraft, class_name: 'Aircraft'
  has_many :AirworthinessDirective, class_name: 'AirworthinessDirective'
  has_many :ServiceBulletin, class_name: 'ServiceBulletin'

end
